### PR TITLE
add jquery global to env

### PR DESCRIPTION
**Pull request by rowbi-wan**
## Suggested rule/changes(s):
 title says it all. currently getting issues with `'$' is not defined` all over the place. 

@netsells/frontend - Please review

### CODE REVIEW

**Comment by samturrell** [comment]
I don't think that's right

**Comment by rowbi-wan** [comment]
@samturrell happy now?

**Comment by samturrell** [comment]
`npm run lint`

File: index.js [hunk 1]
```diff
@@ -9,7 +9,10 @@ module.exports = {
     },
     env: {
         es6: true,
-        node: true,
+        node: true
```
Comment (samturrell) [inline index.js]: comma

File: index.js [hunk 2]
```diff
@@ -9,7 +9,10 @@ module.exports = {
...
+    },
+    globals: {
+        $: true
```
Comment (samturrell) [inline index.js]: comma

**Review by samturrell — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "eslint ./rules/**/*.js"
+    "lint": "eslint ./ndex.js ./rules/**/*.js --fix"
```
Comment (samturrell) [inline package.json]: `ndex`

**Review by samturrell — DISMISSED** [review]

**Comment by samturrell** [comment]
Just checked the docs for this and i think you were actually right the first time...

File: index.js [hunk 4]
```diff
@@ -11,7 +11,10 @@ module.exports = {
         es6: true,
         node: true,
     },
+    globals: {
+        jquery: true,
```
Comment (samturrell) [inline index.js]: should be in env 🤦‍♂️

**Review by samturrell — CHANGES_REQUESTED** [review]

**Review by samturrell — APPROVED** [review]

**Pull request merged**